apollo.sh

```diff
@@ -72,6 +72,12 @@ function apollo_env_setup() {
     APOLLO_ENV="${APOLLO_ENV} USE_ESD_CAN=${USE_ESD_CAN}"
     # Add more here ...
 
+    if [ ! -f "${APOLLO_ROOT_DIR}/.apollo.bazelrc" ]; then
+        env ${APOLLO_ENV} bash "${APOLLO_ROOT_DIR}/scripts/apollo_config.sh" --noninteractive
+    fi
+}
+
+function print_env() {
     info "Apollo Environment Settings:"
```

Comment:
Although I would suggest keep the `printing env summary` section, moving  the "apollo_config.sh" stuff before printing environ summary is meaningful.